.stickler.yml

```diff
@@ -0,0 +1,17 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './eslint.config'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+      - 'node_modules/*'
+      - '.eslintrc.js'
```

Comment:
- Since we are deleting that file we can remove this line.

Reply:
resolved by deleting line .eslintrc.js